resources/js/bots/transactor/bot.js

```diff
@@ -41,15 +28,26 @@ function amountSendParam() {
     return {
         name: "amount",
         type: status.types.NUMBER,
-        suggestions: amountParameterBox.bind(this)
+        placeholder: I18n.t('amount_placeholder')
     };
 }
 
-var paramsPersonalSend = [amountSendParam()];
+var paramsPersonalSend = [assetSendParam, amountSendParam()];
 var paramsGroupSend = [recipientSendParam, amountSendParam()];
 
 function validateSend(validateRecipient, params, context) {
 
+    var allowedAssets = context["allowed-assets"];
+    var asset = params["asset"];
+
+    if(allowedAssets.indexOf(asset) === -1){
```

Comment:
Just curious, if there a reason to do validation here but not on clojure side?

Reply:
if we still have plans to provide some bot API for third parties that would be the reason

Reply:
The more immediate reason is that we need the validation for message as well as wallet, and this is the first place in the flow where we can do this.